webapp/lib/view.dart

```diff
@@ -25,17 +25,31 @@ void init() {
 class NavView {
   DivElement navViewElement;
   DivElement _appLogos;
+  DivElement _navLinks;
   DivElement _projectTitle;
   DivElement _projectOrganizations;
   AuthHeaderViewPartial authHeaderViewPartial;
-
+  ButtonElement _backBtn;
+  AnchorElement _allProjectsLink;
 
   NavView() {
     navViewElement = new DivElement()
       ..classes.add('nav');
     _appLogos = new DivElement()
       ..classes.add('nav__app-logo')
       ..append(new ImageElement(src: 'assets/africas-voices-logo.svg'));
+    _backBtn = new ButtonElement()
+      ..classes.add('nav-links__back-btn')
+      ..text = '< Back'
+      ..onClick.listen((_) => window.history.back());
```

Comment:
I don't think we want to duplicate the browse functionality in this way.
Where we can go from the navbar will need to be specified based on where we are in the app. So if we're on a project dashboard, we want to be able to go to all projects. If we're on the package configuration, we want to go to the project dashboard. Previous page may not always be that page.

Let's remove the the back button and keep the all projects link for now. I've opened issue #40 to track this, best to work on it separately as it'll require I think changing the controller to support passing in which link to show in the navbar, and so defining a hierarchy of pages

Reply:
Noted